src/Discord/Parts/Channel/Message.php

```diff
@@ -210,13 +212,19 @@ public function getMentionsAttribute()
      *
      * @return Member|User The member that sent the message. Will return a User object if it is a PM.
      */
-    public function getAuthorAttribute()
+    public function getAuthorAttribute($type = 0)
```

Comment:
I'm thinking of possibly splitting the `author` attribute into two attributes, one for the User part and one for the Member part but I will come back to it later.